app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -13,284 +13,224 @@
-                        @selected={{@model.administrativeUnit.scope}}
-                        @onChange={{fn (mut @model.administrativeUnit.scope)}}
-                        @triggerId="administrative-unit-scope"
-                        as |scope|
-                      >
-                        {{scope}}
-                      </PowerSelectMultiple>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div class="au-o-grid">
-                {{#each @model.administrativeUnit.identifiers as |identifier|}}
-                  <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                    <div class="au-o-grid">
-                      <div class="au-o-grid__item au-u-1-4">
-                        <p><strong>{{identifier.idName}}</strong></p>
-                      </div>
-                      <div class="au-o-grid__item au-u-3-4">
-                        <p>{{identifier.structuredIdentifier.localId}}</p>
-                      </div>
-                    </div>
-                  </div>
-                {{/each}}
-              </div>
-            </c.content>
-          </AuCard>
-        </div>
+                  <AuControlRadio
+                    @label="Nee"
+                    @name="is-cross-border"
+                    @value={{@model.administrativeUnit.crossBorder}}
+                    @identifier="cross-border-false"
+                    @onChange={{fn
+                      (mut @model.administrativeUnit.crossBorder)
+                      false
+                    }}
+                    checked={{not @model.administrativeUnit.crossBorder}}
+                  />
+                </:content>
+              </Item>
+            </Grid>
+            {{/if}}
+            <Grid class="au-u-margin-top-large" as |Item|>
+              <Item @labelFor="administrative-unit-scope">
+                <:label>Werkingsgebied</:label>
+                <:content>
+                  {{! TODO Make this work }}
+                  <PowerSelectMultiple
+                    @disabled={{true}}
+                    @allowClear={{true}}
+                    @searchEnabled={{true}}
+                    @loadingMessage="Aan het laden..."
+                    @noMatchesMessage="Geen resultaten"
+                    @searchMessage="Typ om te zoeken"
+                    @selected={{@model.administrativeUnit.scope}}
+                    @onChange={{fn (mut @model.administrativeUnit.scope)}}
+                    @triggerId="administrative-unit-scope"
+                    as |scope|
+                  >
+                    {{scope}}
+                  </PowerSelectMultiple>
+                </:content>
+              </Item>
+              {{#each @model.administrativeUnit.identifiers as |identifier|}}
+                <Item>
+                  <:label>{{identifier.idName}}</:label>
+                  <:content>
+                    <p>{{identifier.structuredIdentifier.localId}}</p>
```

Comment:
`<p>` tag isn't needed I think. I do wonder if we actually need to show this on the edit page since you can't edit it.

Reply:
ops

Reply:
Actually, only the KBO number could need to change because it doesn't exist in some cases, but I don't know if it will be fixed in the data source. Need to be confirmed.